lib/synapse-plugins/system-managment.vala

```diff
@@ -466,5 +484,18 @@ namespace Synapse {
 
             return result;
         }
+
+        private bool match (SystemAction action, Gee.Map.Entry<Regex, int> matcher) {
+            if (matcher.key.match (action.title)) {
+                return true;
+            }
+            foreach (var keyword in action.keywords) {
```

Comment:
Missing empty line after if clause

Reply:
Good one, I've -re-checked the whole file for those.